ledger/participant-integration-api/src/main/scala/platform/store/FlywayMigrations.scala

```diff
@@ -32,14 +36,16 @@ private[platform] class FlywayMigrations(jdbcUrl: String)(implicit loggingContex
     }
 
   def migrate(
-      allowExistingSchema: Boolean = false
+      allowExistingSchema: Boolean = false,
+      enableExperimentalSchema: Boolean = false,
   )(implicit resourceContext: ResourceContext): Future[Unit] =
     dataSource.use { ds =>
       Future.successful {
         val flyway = configurationBase(dbType)
           .dataSource(ds)
           .baselineOnMigrate(allowExistingSchema)
           .baselineVersion(MigrationVersion.fromVersion("0"))
+          .target(if (enableExperimentalSchema) "latest" else lastStableSchemaVersion)
```

Comment:
We are assuming we won't need to add a new iteration to the 'stable' schema, right?

Reply:
Adding a new migration for the generic index is still possible:
1. current last stable schema version is 49
2. increase versions for existing experimental schema migrations by 1
3. add migration for generic index schema as version 50
4. set `lastStableSchemaVersion` to 50

Reply:
That makes sense, assuming we don't guarantee that an 'append-only' schema may be migrated after a 'stable' schema change (which is a perfectly reasonable assumption).

Reply:
@gerolf-da  @miklos-da  Please note: as of this is getting merged, the generic indexer DB schema is frozen, until the gradual rollout finishes with the cleanup step. So the step Gerolf is laying out only apply for a change which is a resolution of a major incident impacting all clients or something ;)
Also please note: this is temporary code, as soon as cleanup conducted, these safeguards will disappear.